src/Chapter1.hs

```diff
@@ -632,7 +644,24 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int  
+sumLast2 n = div lastTwoDigits 10 + mod lastTwoDigits 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod x 10
div x 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod ... 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Thanks! `divMod` is a cool thing to know! I don't think I can come up with a shorter answer with `divMod` ...  :( The following is what I came up with.

```
sumLast2 :: Int -> Int
sumLast2 n =
 let (secondLast, last) = divMod n 10
 in (mod secondLast 10) + last
```